README.md

```diff
@@ -0,0 +1,32 @@
+# squashdelta
+Create efficient deltas (patches) between two SquashFS images
+
+## Building from source
+This project uses [autotools](http://inti.sourceforge.net/tutorial/libinti/autotoolsproject.html)
+
+```bash
+$ git clone https://github.com/mgorny/squashdelta.git
+$ cd squashdelta
```

Comment:
There's little activity in git, and there is a proper release .tar (in 'Releases') which is strongly recommended over it.